festim/hydrogen_transport_problem.py

```diff
@@ -98,19 +95,54 @@ def initialise(self):
         """
 
         self.define_function_space()
-        (
-            self.facet_tags,
-            self.volume_tags,
-            self.dx,
-            self.ds,
-        ) = self.mesh.create_measures_and_tags(self.function_space)
+        self.define_markers_and_measures()
         self.assign_functions_to_species()
         self.create_formulation()
 
     def define_function_space(self):
         elements = ufl.FiniteElement("CG", self.mesh.mesh.ufl_cell(), 1)
         self.function_space = fem.FunctionSpace(self.mesh.mesh, elements)
 
+    def define_markers_and_measures(self):
+        dofs_facets, tags_facets = [], []
+
+        # TODO this should be a property of mesh
+        fdim = self.mesh.mesh.topology.dim - 1
+        vdim = self.mesh.mesh.topology.dim
+
+        # find all cells in domain and mark them as 1
```

Comment:
```suggestion
        # find all cells in domain and mark them as 0
```

You seem to be marking them as zero here